frontend/containers/discover-map/map-help/component.tsx

```diff
@@ -0,0 +1,207 @@
+import { FC, useState } from 'react';
+
+import { HelpCircle, Layers, MapPin, Search, ZoomIn, ZoomOut, X as XIcon } from 'react-feather';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import Button from 'components/button';
+import Icon from 'components/icon';
+import Modal from 'components/modal';
+import { theme } from 'tailwind.config';
+
+import clusterExemple from 'svgs/map/cluster-exemple.svg';
+
+import { MapHelpProps } from '.';
+
+export const MapHelp: FC<MapHelpProps> = () => {
+  const { formatMessage } = useIntl();
+  const [open, setOpen] = useState(false);
+
+  return (
+    <div className="">
+      <Button
+        className="flex items-center justify-center w-8 h-8 px-1.5 py-1.5 text-gray-800 bg-white rounded shadow focus-visible:outline-green-dark"
```

Comment:
For consistency with the other buttons on the map, I would change the classes to:
```suggestion
        className="flex items-center justify-center w-8 h-8 px-2 py-1 text-gray-800 bg-white rounded shadow-sm outline-none border-xl focus-visible:ring-green-dark focus-visible:ring-2 hover:ring-green-dark hover:ring-1"
```